wagtailmodelchooser/templates/wagtailmodelchooser/modal.html

```diff
@@ -11,13 +11,19 @@
 <div class="nice-padding">
 	{# Need to keep the form in the HTML, even if the snippet is not searchable #}
 	{# This is to allow pagination links to be generated from the form action URL #}
-	<form class="search-bar" action="{{ chooser_url }}" method="GET">
+	{% block search_form %}
+	<form class="search-bar" action="{{ chooser_url }}" method="GET" style="margin-bottom:20px">
```

Comment:
Inline styles aren't ideal, I think from memory this used to pretty similar to the current wagtail modal, is there any classes we can leverage from there? I'm not too fussed if not

Reply:
Copied some HTML/classes from the image selector modal. No more inline styles!